declarative_clusterconfig/configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc

```diff
@@ -13,7 +13,9 @@ With {gitops-title}, you can configure Argo CD to recursively sync the content o
 
 * You have logged in to the {OCP} cluster as an administrator.
 * You have installed the {gitops-title} Operator in your cluster.
-* You have logged into Argo CD instance.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the {gitops-title} `argocd` CLI.
+* You have logged into Argo CD instance. 
```

Comment:
Can we change this to `You have logged into OCP cluster`. Because logging into ArgoCD is required only in the normal mode and we have given instructions on how to login. In core mode, user's may not have to be logged in to ArgoCD instance.